### PR TITLE
Add onLoad event to CldImage documentation

### LINKED ISSUE #497: [Feature] ADd documentation for onLoad event for CldImage
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

https://youtu.be/rW-xDs8HU-o?t=1609

Had trouble finding onLoad in documentation. Should have an events section. This is an inherited Next.js prop

Rather than re-explaining every prop, mention in the events section that the events are inherited and point to the Next.js docs.

### PR DESCRIPTION
**Pull request by mr-loop-1**
# Description

## Issue Ticket Number

Fixes #497 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/cloudinary-community/next-cloudinary/issues) ticket for this PR
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/cloudinary-community/next-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have run tests locally to ensure they all pass
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: docs/pages/cldimage/configuration.mdx [hunk 1]
```diff
@@ -1933,13 +1939,26 @@ until it's available, then force refresh the image instance in the DOM.
 **Examples**
 
 ```jsx copy
-onError={() => {
+onError={(e) => {
```
Comment (colbyfayock) [inline docs/pages/cldimage/configuration.mdx:1942]: does this actually pass an error/event? i thought this just passed back the element itself

File: docs/pages/cldimage/configuration.mdx [hunk 2]
```diff
@@ -1933,13 +1939,26 @@ until it's available, then force refresh the image instance in the DOM.
...
 [Learn more about onError](https://nextjs.org/docs/pages/api-reference/components/image#onerror) on the Next.js docs.
 
+#### `onLoad`
+
+Callback that fires when an image loads completely and any [placeholder](/guides/placeholders) has been removed.
+
+**Examples**
+
+```jsx copy
+onLoad((e) => {
```
Comment (colbyfayock) [inline docs/pages/cldimage/configuration.mdx:1956]: similar question about the argument here

**Comment by colbyfayock** [comment]
thanks @mr-loop-1 this is great, just have a question there for you

File: docs/pages/cldimage/configuration.mdx (hunk 1 above)
Comment (mr-loop-1) [reply docs/pages/cldimage/configuration.mdx:1942]: Actually it does pass event object, this event has a property called target which contains the pointer to element.

Comment (colbyfayock) [reply docs/pages/cldimage/configuration.mdx:1942]: ah just double checked in my code and you're right, for some reason though it only passed the element. thanks for confirming!

**Comment by colbyfayock** [comment]
thanks @mr-loop-1 - this PR qualifies for free Cloudinary Hacktobefest swag! Please send me an email at hacktoberfest@cloudinary.com with your name, GitHub username, and a link to this PR where I'll send you further instructions. Happy Hacktoberfest!

**Comment by colbyfayock** [comment]
@all-contributors please add @mr-loop-1 for doc

**Pull request merged**